src/pytest_postgresql/factories/process.py

```diff
@@ -25,11 +25,11 @@
 import pytest
 from _pytest.fixtures import FixtureRequest
 from _pytest.tmpdir import TempdirFactory
+from port_for import get_port
```

Comment:
There is a difference in types, so we might need

```
@@ -104,7 +104,7 @@ def postgresql_proc(
                 DeprecationWarning,
             )
 
-        pg_port = get_port(port) or get_port(config["port"])
+        pg_port = get_port(port) or get_port(config["port"])  # type: ignore[arg-type]
         assert pg_port is not None
         datadir = tmpdir.mkdir(f"data-{pg_port}")
         logfile_path = tmpdir.join(f"{logfile_prefix}postgresql.{pg_port}.log")
```

Reply:
Hmmm... one thing I noticed is that the port_for does not allow for None's 🤔

Reply:
https://github.com/kmike/port-for/pull/28